CppCustomVisualizer2/README.md

```diff
@@ -0,0 +1,15 @@
+## C/C++ Custom Visualizer
```

Comment:
Please update this readme for this specific visualizer scenario.

Reply:
Will update once the wiki page (sent via email) is created.